rfcs/text/0009_screenshot_mode_service.md

```diff
@@ -0,0 +1,149 @@
+- Start Date: 2020-03-02
+- RFC PR: (leave this empty)
+- Kibana Issue: (leave this empty)
+
+# Summary
```

Comment:
Is reporting still just in Discover, Dashboard, and Canvas, or are there any other places where we use it? (Found that list from our docs, but might be nice to outline which apps support reporting today in this RFC).

Reply:
Yes, it's in Visualize Editor as well.

Let's make Reporting better so this list goes from 3 to 300!